backend/src/services/userService.ts

```diff
@@ -0,0 +1,95 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import crypto from 'crypto';
+import { logger } from './../utils/logger';
+import { TokenPayload } from '../types/TokenPayload';
+
+/**
+ * The secret used for signing the web tokens.
+ * If we want the secret to remain same over runtimes
+ * (meaning that the te tokens from previous runtime will still be valid)
+ * the JWT_SECRET environment variable should be defined.
+ * The secret must be cryptographically high quality random string.
+ * If an attacker can guess the secret they can sign their own tokens.
+ */
+const secret =
+    process.env.JWT_SECRET || crypto.randomBytes(32).toString('base64');
```

Comment:
I think we should just check process.env.JWT_SECRET and add a development token to .env-sample, and then verify it's set during startup